pyanaconda/modules/storage/partitioning/custom_partitioning.py

```diff
@@ -324,8 +324,7 @@ def _execute_partition_data(self, storage, data, partition_data):
                     lineno=partition_data.lineno
                 )
 
-            should_clear = storage.should_clear(disk)
-            if disk and (disk.partitioned or should_clear):
+            if disk and disk.partitioned:
```

Comment:
@vojtechtrefny, we run this code after the disks are cleared and initialized, so I think that it should be enough to check the `partitioned` property. What do you think?

Reply:
`partitioned` just checks if the disk is formatted to a "disklabel", so it should be enough to check only `partitioned` after disk initialization.